### PR TITLE
Tab basic styles

### PR DESCRIPTION
**Pull request by Charles-ErickBG**
### Proposed Changes

- Reduce paragraph length and increase line height to improve readability.
- Make default font size inside tabs bigger.
- Fix margins so that titles are closer to the content they refer to (design rule of proximity).

### Potential Breaking Changes

None

### Acceptance Criteria

None

### CODE REVIEW

**Comment by Charles-ErickBG** [comment]
Screen Shot 2019-09-23 at 3 45 47 PM

File: packages/react-vapor/docs/src/OneDemoToRuleThemAll.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ function TopNav() {
 function Header() {
     return (
         <div id="header" className="header flex flex-center space-between p2" style={{minHeight: '65px'}}>
-            <h1 className="h1">
+            <h1 className="h1 text-white">
```
Comment (francoislg) [inline packages/react-vapor/docs/src/OneDemoToRuleThemAll.tsx]: Why is this `text-white` required? Should the default styling be correct for this already? :P

File: packages/react-vapor/src/components/button/examples/ButtonExamples.1.use.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+| Isolated  | Isolated Action Buttons are used for extra actions located on their own within an interface. Button borders and background color are removed to reduce the noise generated by the presence of many consecutive borders and surfaces, or to make it more discrete in the screen. The icon is optional and aims to help users identify the action faster. However, keep in mind that too many icons in a screen can be as distracting and confusing as too less.                             |
+| Icon Only | Icon Only Buttons are used when there is a lack of space and a high density of content. This button type is preferably used to represent labels that are easily represented by an icon, such as the word "More".                                                                                                                                                                                                                                                                           |
+
+---
+
+## Related Components
+
+[Action Bar](../#/components/ActionBar)
```
Comment (GermainBergeron) [inline packages/react-vapor/src/components/button/examples/ButtonExamples.1.use.md]: Links should start from the hash

```suggestion
[Action Bar](#/components/ActionBar)
```

File: packages/react-vapor/src/components/button/examples/ButtonExamples.1.use.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+["More" Menu Button](../#/components/Menu)
```
Comment (GermainBergeron) [inline packages/react-vapor/src/components/button/examples/ButtonExamples.1.use.md]:
```suggestion
["More" Menu Button](#/components/Menu)
```

File: packages/react-vapor/src/components/button/examples/ButtonExamples.1.use.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+[Tabs](../#/components/Tabs)
```
Comment (GermainBergeron) [inline packages/react-vapor/src/components/button/examples/ButtonExamples.1.use.md]:
```suggestion
[Tabs](#/components/Tabs)

```

File: packages/vapor/scss/typography/headings.scss [hunk 5]
```diff
@@ -4,38 +4,59 @@ h3,
...
+    margin-top: 1.5em;
     font-size: 13px;
 }
 
 h6 {
+    margin-top: 1.5em;
     font-size: 13px;
 }
 
+#header {
```
[the remark this answers is not in the document]
Comment (Charles-ErickBG) [reply packages/vapor/scss/typography/headings.scss]: This is for the name in the top navigation of the site.
The ID already was #header.

Should I change it or it is ok?

**Comment by kpoulincoveo** [comment]
Because of the padding of the section, the first H1 title seems to have too much space. Maybe we should remove the padding since we'll always start with a H1 title and increase the spacing before H1 title a little more?
image

Also, the divide line seems odd because it is under a table, but I guess we will have to live with it.

Everything else seems fine though

**Pull request merged**